schema_utils_test.py

```diff
@@ -198,7 +198,15 @@
 
 
 def _validate_ui_config(obj_type, ui_config):
-    """Validates the value of a UI configuration."""
+    """Validates the value of a UI configuration.
+
+        Args:
+            obj_type: str. UI config spec type.
+            ui_config: dict. The UI config that needs to be validated.
+
+        Raises:
+            AssertionError. The object fails to validate against the schema.
```

Comment:
Are we validating the object in this function? (Ditto elsewhere, the Raises message for other docstring looks wrong (not sure!))

Reply:
@DubeySandeep So both the functions call `normalize_against_schema` from `schema_utils.py`. This function throws an Assertion error which says `The object fails to validate against the schema.`. That is why this message is written here. Is that wrong?